dist/stat_data.py

```diff
@@ -245,6 +264,25 @@ def __init__(self, name, desc, flags=''):
     DhandleStat('dh_sweeps', 'connection sweeps'),
 
     ##########################################
+    # Locking statistics
+    ##########################################
```

Comment:
I would suggest starting with cache->evict_queue_lock, cache->evict_walk_lock and queue->evict_lock.  We can run our eviction tests, look at stats and timeseries and reassess if we want to stop tracking any of those.